site/src/img/down-chevrons.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="13px" height="16px" viewBox="0 0 13 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 48.2 (47327) - http://www.bohemiancoding.com/sketch -->
+    <title>arrow</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Homepage-1280px" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(-634.000000, -1258.000000)">
```

Comment:
@jackturner - Can you strip the ID attributes out of this SVG?  When it gets embedded in the page multiple times, it causes the same ID to be used multiple times, so I've just be stripping ids (and classes where they're not strictly necessary) out of SVGs manually.

Reply:
Will do, didn't notice it.

Reply:
Done.